scripts/force_renew_allocation_sources.py

```diff
@@ -0,0 +1,13 @@
+import django
+
+django.setup()
+
+from cyverse_allocation.tasks import renew_allocation_sources
+
+
+def main():
+    renew_allocation_sources(renewal_strategy='default', ignore_current_compute_allowed=True, dry_run=False)
```

Comment:
Maybe add some `argparse` to the script?
```
    parser = argparse.ArgumentParser()
    parser.add_argument("--dry-run", action="store_true",
                        help="Print output rather than perform operation"
```

Reply:
Good idea.